pytorch_lightning/utilities/seed.py

```diff
@@ -66,3 +67,17 @@ def seed_everything(seed: Optional[int] = None) -> int:
 
 def _select_seed_randomly(min_seed_value: int = 0, max_seed_value: int = 255) -> int:
     return random.randint(min_seed_value, max_seed_value)
+
+
+def pl_worker_init_function(worker_id: int) -> None:
+    """
+    The worker_init_fn that Lightning automatically adds to your dataloader if you previously set
+    set the seed with :func:`~pytorch_lightning.utilities.seed.seed_everything`.
+
+    See Also
+        `Randomness in DataLoaders <https://pytorch.org/docs/stable/notes/randomness.html#dataloader>`_
+    """
+    #
+    worker_seed = torch.initial_seed() % (2 ** 32)
+    numpy.random.seed(worker_seed)
+    random.seed(worker_seed)
```

Comment:
```suggestion
    worker_seed = (torch.initial_seed() + worker_id) % (2 ** 32)
    numpy.random.seed(worker_seed)
    random.seed(worker_seed)
```
probably need to add this worker_id, not sure why pytorch docs don't have it